Description: reduce debug logging level
Author: Jens Peter Secher <jps@debian.org> & Peter Marschall

## This code was put into the public domain by Jens Peter Secher <jps@debian.org>.

# Peter Marschall: refreshed against 1.97 & renamed to reduce-debug-logging.diff

--- a/authfile.c
+++ b/authfile.c
@@ -602,7 +602,7 @@ key_parse_private_pem(Buffer *blob, int
 		EVP_PKEY_free(pk);
 	if (prv != NULL && commentp)
 		*commentp = xstrdup(name);
-	debug("read PEM private key done: type %s",
+	debug2("read PEM private key done: type %s",
 	    prv ? key_type(prv) : "<unknown>");
 	return prv;
 }
